### DISCUSSION TITLE
Create new referential (ex. ISO/IEC 27002:2022)

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#412] [Ideas]
Hi all,

I'would like to create a new referential from PCI-DSS, ISO 27017, ISO 27701 However I don't find such option in the documentation.

Or .... It should be great, to have the capability to record a cross reference to other referential, using ISO 27002 for example.

A functionality such as:

ISO 27002          |          PCI-DSS
§12.4.2               |          Req1 1.2.3.4
§14.5.6               |          Req4 4.5.3.6

Regards,

**Comment by jerolomb** [comment]
Hello,

You can create new referential [1] in the knowledge base [2]. You can also cross reference the reference the referential. We call that a mapping (the withe 2 on red background on documentation). 

Don't hesitate to come back to us if it's not clear,

Regards 

[1] https://www.monarc.lu/documentation/user-guide/#referentials
[2] https://www.monarc.lu/documentation/user-guide/#management-of-knowledge-base

**Reply by ruslanbaidan** [reply]
Hello,

What also might be interesting for you is the Statement of Applicability. You can find the short description here:
https://www.monarc.lu/documentation/user-guide/#statement-of-applicability

To be noted, not all the latest functionality is described in the user guide. We would recommend you to try Monarc out by using a VM. You would be able to test and find out how far it meets your expectations:
https://www.monarc.lu/download/#virtual-machine

With best regards.

**Reply by danje57** [reply]
Thanks to both of you,

I'll try it this weke end and revert back to you.

Hope it will be a benefit for the community... at least if I success :p

**Reply by danje57** [reply]
ello all,

Sorry, I don't forget that I need to create the referential.
However, I'm waiting for ISO 27002:2022, as 27701 use some controls of it.

Regards

**Reply by jerolomb** [reply]
Hello,

You can import iso27002:20022 https://www.youtube.com/watch?v=N45p3fw_G2U 

We don't have 27701 in MONARC for the moment you can create it in the KB and also create a mapping between the two referential. 

Can you describe your issue in details ? Or do you need more help to create ? 

Regards

**Reply by danje57** [reply]
No issues on my side, only missing time to make all the stuff. ;-)

**Reply by danje57** [reply]
However Video tutorial works fine!!! Thanks again!!
